index.js

```diff
@@ -42,7 +42,7 @@ module.exports = {
         'vendor/ember-cli-mocha/test-loader.js'
       ];
 
-      var addonOptions = app.options['ember-cli-mocha'];
+      var addonOptions = app.options['ember-cli-mocha'] || {};
       if (addonOptions && !addonOptions.disableContainerStyles) {
```

Comment:
Not sure why the initialization to `{}` is needed. Won't the first check for the truthiness of `addonOptions` cover the use case in which the option doesn't exist?